src/openrct2/cheats.c

```diff
@@ -428,6 +430,60 @@ static void cheat_set_staff_speed(uint8 value)
 	}
 }
 
+static void cheat_own_all_land()
+{
+	sint32 min = 32, max = gMapSizeUnits - 32;
+	for (sint32 y = min; y <= max; y += 32) {
+		for (sint32 x = min; x <= max; x += 32) {
+			rct_map_element* surfaceElement = map_get_surface_element_at(x / 32, y / 32);
+
+			// Ignore already owned tiles
+			if (surfaceElement->properties.surface.ownership & OWNERSHIP_OWNED ||
+				surfaceElement->properties.surface.ownership & OWNERSHIP_CONSTRUCTION_RIGHTS_OWNED)
```

Comment:
I think you should change this check to
```
if (surfaceElement->properties.surface.ownership & OWNERSHIP_OWNED)
```
That way, existing construction rights can be fully owned as well, providing they qualify.